tests/unit/modules/kubernetes_test.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email:`Jochen Breuer <jbreuer@suse.de>`
+'''
+
+# Import Python Libs
+from __future__ import absolute_import
+import os
+
+# Import Salt Testing Libs
+from salttesting import TestCase, skipIf
+from salttesting.mock import (
```

Comment:
This import also needs to be updated to `from tests.support.mock import (`

We moved a bunch of the salttesting libs around for 2017.7 and onward.